bindings/c/test/mako/mako.cpp

```diff
@@ -62,6 +62,8 @@ namespace mako {
 struct alignas(64) ThreadArgs {
 	int worker_id;
 	int thread_id;
+	bool tenant_main;
+	int tenants;
```

Comment:
Does each thread map to exactly one tenant?

Reply:
No, there can be more (or fewer) tenants than threads